index.html

```diff
@@ -90,7 +90,8 @@ <h2>Year: <label id='active-year'></label></h2>
 			'fill-color': '#267300',
 			},
 //		'filter': ['all', Year_Prot]
-	});
+	}, 'road-simple'
```

Comment:
This line is the only important change here, everything else is just my text editor being finicky about whitespace.  It's very simple, I just had to look up the actual syntax: after the {object} that defines the layer you're adding, you can just list the ID of the existing layer to add it behind.